### PR TITLE
fix(chat): remove horizontal scroll (Issues #1095, #1092)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

 Remove horizontal scroll 

Issues:

- https://github.com/epam/ai-dial-chat/issues/1095
- https://github.com/epam/ai-dial-chat/issues/1092

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1097.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1097.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/ChatMessage/AdjustedTextarea.tsx [hunk 1]
```diff
@@ -41,10 +59,10 @@ export const AdjustedTextarea = React.forwardRef((props: Props, ref) => {
         value={value}
       />
       <textarea
+        {...restProps}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ChatMessage/AdjustedTextarea.tsx]: Please, revert this to previous place to not override custom `data-qa` (reason of failed e2e-tests)

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1097.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**